src/utils.jl

```diff
@@ -23,12 +23,26 @@ end
 function getcrs(table)
     if GeoInterface.isfeaturecollection(table)
         return GeoInterface.crs(table)
-    elseif first(DataAPI.metadatasupport(typeof(table)))
+    end
+    crs = geomcrs(table)
+    if !isnothing(crs)
+        return crs
+    end
+    if first(DataAPI.metadatasupport(typeof(table)))
         crs = DataAPI.metadata(table, "GEOINTERFACE:crs", nothing)
         if isnothing(crs) # fall back to searching for "crs" as a string
             crs = DataAPI.metadata(table, "crs", nothing)
         end
         return crs
+    end
+    nothing
+end
```

Comment:
This would be great to upstream into GeoInterface, maybe if we make it a bit more generic to Tables.jl geometries?

Reply:
Yes, ideally as a package extension on DataAPI?

Reply:
DataAPI is already a dependency i think (but maybe that PR is still open